src/pages/Registration.js

```diff
@@ -12,33 +14,43 @@ export default function Registration() {
 
   function onSubmit(formData) {
     console.log(formData)
-    auth
+    authentication
       .createUserWithEmailAndPassword(formData.email, formData.password)
-      .then((response) => {
-        console.log(response)
+      .then((firebaseResponse) => {
+        console.log(firebaseResponse)
       })
   }
   return (
     <>
       <StyledMain>
-        <h1>Registriere dich hier:</h1>
         <Formik
           initialValues={{ email: '', password: '' }}
           validationSchema={validationSchema}
           onSubmit={onSubmit}
         >
-          {({ isSubmitting }) => (
-            <FormStyled>
-              <FieldStyled type="email" name="email" placeholder="E-Mail" />
+          {({ isSubmitting, values, handleSubmit, handleChange }) => (
+            <FormStyled onSubmit={handleSubmit}>
```

Comment:
Ich kenne mich mit Formik nicht aus, aber macht handleSubmit hier etwas? Ich kann die Funktion sonst niergends wiederfinden...

Reply:
jaa das ist so eine vorgabe von formik, dadurch kommen die daten in firebase an 🤔